cfpb/consumerfinance.gov
# financial well-being wonky no-js

**Taken from Financial well-being has wonky no JS logic #4504**

If you visit https://www.consumerfinance.gov/consumer-tools/financial-well-being/ the "Get my score" button at the bottom is disabled. This disabling is controlled through JavaScript. So, if JavaScript is disabled, that button will be enabled. If you then click it with JS disabled, you end up on https://www.consumerfinance.gov/consumer-tools/financial-well-being/error/

This seems wrong. It doesn't seem like you should allow the user to fill out the form, if they are presented an error when it is submitted. It seems like we should just show a notification when JS is disabled, like is done in OAH and ROUT (see https://github.com/cfpb/consumerfinance.gov/pull/7557, for an example).

To test:
Disable JS, and reload page to see removal of form and error popup on http://localhost:8000/consumer-tools/financial-well-being/

Disable JS in chrome: https://developer.chrome.com/docs/devtools/javascript/disable/
